Cargo.toml

```diff
@@ -10,8 +10,12 @@ authors = ["MaidSafe Developers <dev@maidsafe.net>"]
 edition = "2018"
 
 [dependencies]
-threshold_crypto = "~0.3.1"
+hex = "~0.3.1"
+maidsafe_utilities = "~0.17.0"
+rand = "~0.4.2"
```

Comment:
Should be in `[dev-dependencies]`. And probably we can use `~0.6.4` here?

Reply:
Looks like `rand = "~0.6.4"` doesn't work because maidsafe_utilities 0.18.0 uses 0.3.15:

```
error[E0599]: no method named `gen_iter` found for type `maidsafe_utilities::seeded_rng::SeededRng` in the current scope
   --> src/immutable_data.rs:159:25
    |
159 |         let value = rng.gen_iter().take(len).collect();
    |                         ^^^^^^^^
    |
    = help: items from traits can only be used if the trait is in scope
    = note: the following trait is implemented but not in scope, perhaps add a `use` for it:
            `use rand::Rng;`

warning: unused import: `rand::Rng`
   --> src/immutable_data.rs:111:9
    |
111 |     use rand::Rng;
    |         ^^^^^^^^^
    |
    = note: #[warn(unused_imports)] on by default
```